addons/purchase/static/src/views/purchase_dashboard.scss

```
@@ -3,6 +3,14 @@
         width: 100%;
         height: 100%;
         border-radius: 0;
+
+        &:not(.o_no_hover):hover {
```

There shouldn't be need of such an high specificity
```suggestion
        &:where(:not(.o_no_hover):hover) {
```